### DISCUSSION TITLE
Should ScheduledJob return ITriggerConfigurator

### DISCUSSION THREAD

**Discussion by billrob** [quartznet/quartznet#1663] [Ideas] [answered]
It feels more fluenty that way as chaining on the single parameter passed to delegate is icky.  I know all feels words.  First time setting up quartz in .net core, been doing Framework quartz for years.

**Comment by lahma** [comment, marked as answer]
Can you elaborate a bit more what kind of API you are after? I think we are speaking about this API:

https://github.com/quartznet/quartznet/blob/f64d3c3a400958b8a43ffab0937bed39fc832cf1/src/Quartz.Examples.AspNetCore/Startup.cs#L147-L156

**Reply by billrob** [reply]
I withdraw, I got used to it.
